packages/visx-wordcloud/package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "@visx/wordcloud",
+  "version": "0.1.0",
+  "description": "visx wordcloud",
+  "sideEffects": false,
+  "main": "lib/index.js",
+  "module": "esm/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib",
+    "esm"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/airbnb/visx.git"
+  },
+  "keywords": [
+    "visx",
+    "react",
+    "d3",
+    "visualizations",
+    "charts",
+    "svg"
+  ],
+  "author": "Cezar Craciun @_cezarcraciun",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/airbnb/visx/issues"
+  },
+  "homepage": "https://github.com/airbnb/visx#readme",
+  "peerDependencies": {
+    "react": "^16.8.0-0",
```

we **just** landed `react@17` support, so maybe can open this up to the following? 

```suggestion
    "react": "^16.8.0-0 || ^17.0.0-0",
```